ear/src/main/resources/s3binaries/nxserver/config/s3directupload-config.xml.nxftl

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+
+<component name="s3DirectUpload" version="1.0.0">
+
+<#if "${nuxeo.s3storage.useDirectUpload}" == "true">
+
+  <require>org.nuxeo.ecm.core.storage.cloud.managment.contrib</require>
+
+  <extension target="org.nuxeo.ecm.core.transientstore.TransientStorageComponent" point="store">
+    <store name="s3TransientStore" class="org.nuxeo.ecm.core.transientstore.keyvalueblob.KeyValueBlobTransientStore">
+      <targetMaxSizeMB>1</targetMaxSizeMB>
```

Comment:
This seems a bit small, and we should give admins an easy way to customize this. Please make this a property too.